_kits/it/analisi-contesto.md

```diff
@@ -45,6 +45,9 @@ superAccordion:
         - title: Mappa degli attori
           icon: "/assets/icons/pencil.svg"
           description: Uno schema che rappresenta la rete di relazioni tra l’utente e gli altri attori del contesto
+          url: 'https://docs.google.com/presentation/d/1kxsFXSesX2kCOR1L3SbPfYqFInFhf_A7RW3_2xZ943o/edit?usp=sharing'
+          urlText: 'Vai alla risorsa'
+          urlArialabel: 'Vai alla risorsa (link esterno)'
           link: <a href='https://docs.google.com/presentation/d/1kxsFXSesX2kCOR1L3SbPfYqFInFhf_A7RW3_2xZ943o/edit?usp=sharing' aria-label="Vai alla risorsa (link esterno)" target="_blank">Vai alla risorsa</a>
```

Comment:
```suggestion
```

Reply:
Correct to remove on resources as soon as changed to the new mode.